src/parallel/parallel.cpp

```diff
@@ -56,6 +58,9 @@ ParallelBucket::unlock()
 /// THREAD MANAGER
 ////////////////////////////////////////////////////////////////////////////////
 
+ThreadManager::ThreadManager(class ParallelRobinHoodHashTable *const hash_table)
```

Comment:
```suggestion
ThreadManager::ThreadManager(ParallelRobinHoodHashTable *const hash_table)
```

We had to forward-declare the HashTable class in the header but we shouldn't need to here. Non-blocking.

Reply:
Oh I see. I was confused about that. Thanks for pointing it out.

I take it you can forward declare within another class, e.g.
```cpp
private:
    class ParallelRobinHoodHashTable *const hash_table_;
```

Reply:
Yup you can.